docarray/typing/tensor/audio/audio_ndarray.py

```diff
@@ -0,0 +1,83 @@
+import wave
+from typing import TYPE_CHECKING, BinaryIO, TypeVar, Union
+
+from docarray.typing import NdArray
+
+T = TypeVar('T', bound='AudioNdArray')
+
+if TYPE_CHECKING:
+    from docarray.proto import NodeProto
+
+
+class AudioNdArray(NdArray):
```

Comment:
remove the implementation of both methods and just apply this modification
```suggestion
class AudioNdArray(AudioTensorMixin, NdArray):
```